src/pages/SamplePage/SamplePage.tsx

```diff
@@ -1,11 +1,10 @@
 import React, { Suspense, lazy, useEffect } from 'react';
-import { Link, withRouter } from 'react-router-dom';
-import { useDispatch } from 'react-redux';
-
 import { Button, StackItem, Stack, Title, Spinner } from '@patternfly/react-core';
+import { addNotification } from '@redhat-cloud-services/frontend-components-notifications/redux';
 import { Main } from '@redhat-cloud-services/frontend-components/Main';
 import { PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components/PageHeader';
-import { addNotification } from '@redhat-cloud-services/frontend-components-notifications/redux';
+import { useDispatch } from 'react-redux';
+import { Link, withRouter } from 'react-router-dom';
```

Comment:
not sure but shouldn't these come after 'react' import?

Reply:
Added extra path groups to handle this scenario. By default `@` groups are sorted before normal imports.